--- /opt/OpenvStorage/webapps/api/backend/views/albanodes.py.org	2016-03-22 17:11:58.919888065 +0000
+++ /opt/OpenvStorage/webapps/api/backend/views/albanodes.py	2016-03-22 16:46:17.415832142 +0000
@@ -76,6 +76,7 @@
             node_list._executed = True
             node_list._guids = [node.guid]
             node_list._objects = {node.guid: node}
+            node_list._data = {node.guid: {'guid': node.guid, 'data': node._data}}
             return node_list
 
         nodes = {}
@@ -100,6 +101,7 @@
         node_list._executed = True
         node_list._guids = nodes.keys()
         node_list._objects = nodes
+        node_list._data = {node.guid: {'guid': node.guid, 'data': node._data}}
         return node_list
 
     @log()
