--- /opt/OpenvStorage/ovs/extensions/generic/heartbeat.py.org	2016-04-20 08:14:50.131293884 +0000
+++ /opt/OpenvStorage/ovs/extensions/generic/heartbeat.py	2016-04-20 08:16:28.226060665 +0000
@@ -36,7 +36,7 @@
 for node in routers:
     if node.heartbeats is None:
         node.heartbeats = {}
-    if 'celery@{0}: OK'.format(node.name) in worker_states:
+    if 'celery@{0}: OK'.format(check_output('hostname -f',shell=True).strip()) in worker_states:
         node.heartbeats['celery'] = current_time
     if node.machine_id == machine_id:
         node.heartbeats['process'] = current_time
