--- /opt/OpenvStorage/ovs/lib/setup.py.org	2016-03-24 10:01:12.262876416 +0000
+++ /opt/OpenvStorage/ovs/lib/setup.py	2016-03-24 09:57:52.886865131 +0000
@@ -1321,7 +1321,7 @@
 
             logger.debug('Copying Rabbit MQ cookie')
             contents = master_client.file_read(rabbitmq_cookie_file)
-            master_hostname = master_client.run('hostname')
+            master_hostname = master_client.run('hostname -s')
             target_client.dir_create(os.path.dirname(rabbitmq_cookie_file))
             target_client.file_write(rabbitmq_cookie_file, contents)
             target_client.file_chmod(rabbitmq_cookie_file, mode=400)
